core/shared/utils/bh_log.c

```diff
@@ -38,7 +38,7 @@ bh_log(LogLevel log_level, const char *file, int line, const char *fmt, ...)
     t = t % (60 * 60);
     m = t / 60;
     s = t % 60;
-    mills = (uint32)(usec % 1000);
+    mills = (uint32)(usec / 1000);
```

Comment:
Should not change the code? `s = t % 60` is used to get the second value, here `mills = (uint32)(usec % 1000)` is to get the remainder, it should be smaller than 1 second, or 1000 ms.

Reply:
At the beginning of the function, we got usec(microsecond=10-6 second), in order to get millisecond(10-3 second), we should divide it by 1000 rather than get the remainder of usec(still microsecond), I observed a non-incremental log time like `00:00:00:792` to `00:00:00:101` so I traced back here

Reply:
But it will keep increasing and be larger than 1000 if using `/`. I guess I know why, seems we should use `mills = (uint32)((usec % 1000000) / 1000)` which limits the usec to be in 1 second first.